### ISSUE TITLE
OLS30103 should not be an error

### ISSUE THREAD

**Issue by william-andre** [odoo/odoo-ls#163]
When importing from another addon in the same path, it is not an issue if that addon is not in the dependencies.

The code will not populate the registry, fields won't exist, etc. But the code is still reachable and can be imported using the python `import` keyword without an error.

Maybe not the cleanest code in the end, but I think it should be bumped to an `INFO` instead of an `ERROR`, at least in the case where both modules are in the same addons path.

**Comment by fda-odoo** [comment]
I agree that this is not an 'error' as the code will work, but still. You have a dependency between the two modules and this is not reflected anywhere, except in this import statement.
I personnaly would let it as "error", but I can understand that some actual code is not following that rule, and it can be annoying to see them as errors. 
I'll consider adding an option about this diagnostic (and in a more general way, for all diagnostics), so you'll be able to tweak the tool according your preferences.

**Comment by william-andre** [comment]
An error should only be reported if the code will lead to a runtime error for instance.
Not when there is nothing wrong at all except the code not being the cleanest.

**Comment by mmahrouss** [comment]
The current version allows the user to disable or change the error level of any diagnostic, closing the issue.
